main.cpp

```diff
@@ -21,15 +21,16 @@ int main(int argc, char *argv[])
 	LOG_DEBUG<<"hello world \n";
 
 	hal::HAL hal;
-	logicLayer::test::Test test = logicLayer::test::Test(&hal);
 
-	test.actuatorsTest();
-	test.mmiTest();
-	test.threadSafenessInGpioTest();
-	test.singletonThreadSafeTest();
+	logicLayer::test::Test test = logicLayer::test::Test(&hal);
 
+	//========= Observer Pattern Test ==================
+	logicLayer::LogicLayer loLay = logicLayer::LogicLayer(hal);
+	hal.notify_observers();
```

Comment:
den Test entweder in die Test.h Datei einordnen oder einfach raus nehmen. Das war eher so fürs entwickeln denke ich.

Reply:
done